### ISSUE TITLE
Accessibility: heading elements should be in a sequentially-descending order

### ISSUE THREAD

**Issue by obulat** [creativecommons/chooser#319]
## Problem

The Chooser currently uses the `<h1>`-`<h6>` heading levels according to the design. Headings are used for creating the outline for a page, which is important for accessibility purposes. Skipping heading levels makes the outline, or website's table of contents, incorrect. This also lowers our Lighthouse Accessibility score:
![image](https://user-images.githubusercontent.com/15233243/107942953-d1009b00-6f40-11eb-863d-92b12abbb41f.png)

## Description

Fortunately, CC's design system provides a way to use the headings' visual styles in an accessible way:

To create a second-level heading that visually looks like `<h5>`, we can use `<h2 class="title is-5">`. 

So, this is the list of changes that need to be introduced:

- Main heading ('License Chooser') should be `<h1>` that looks like `h2`
- Section headings (`h3`) should be `<h2>` with the style of `h3`
- Step header headings (`h5`) should be `<h2>` with the style of `h5`

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by Cronus1007** [comment]
@obulat Just a suggestion regarding the labels of new issues to mark their status as ready for work since `awaiting triage` much refers that these issues are still on hold.

**Comment by neeraj-2** [comment]
@obulat I would like to work on it.

**Comment by zackkrida** [comment]
@Cronus1007 @neeraj-2 It was labeled 'awaiting triage' because it _wasn't_ ready for work. Now it is though, so anyone who wants to work on it can!
